doc/usecases/transforming_coordinate_systems.adoc

```diff
@@ -91,19 +91,19 @@ In Open Simulation Interface, an object's position is defined by the coordinates
 This field defines the orientation of the vehicle's reference point in global coordinates.
 
 ``GroundTruth::moving_object::vehicle_attributes::bbcenter_to_rear``::
-This field specifies the vector pointing from the vehicle's reference point to the middle of the rear axle under neutral load conditions in the vehicle coordinates.
+This field specifies the vector pointing from the vehicle's reference point to the middle of the rear axle under neutral load conditions.
 
 ``SensorData::mounting_position``::
-This field defines the sensor's position and orientation and thereby the origin of the sensor coordinate system.
-The mounting position is given in the vehicle coordinate system.
+This field defines the sensor's virtual mounting position and orientation and thereby the origin of the virtual sensor coordinate system.
```

Comment:
Why not just say "mounting position" instead of "virtual mounting position"? Has the virtual some special meaning hiere?

Reply:
I tried to emphasize the distinction between physical and virtual mounting positions so that they do not get confused. The SensorData::mounting_position is virtual/"imaginary", while SensorDetectionHeader::mounting_position is referring to the actual physical mp of a detector. The OSI class reference docs also describe the SensorData::mounting_position as "virtual mounting position".
I just tried to align/unify terms so far. But I think it would make sense to also add an explanation of the physical mounting position in this section.

Reply:
Just as a side-note: For OSI 4.0 it would be a good idea to name the fields accordingly.